### PR TITLE
Removed "this->"

### PR DESCRIPTION
**Pull request by Founderroni**
* Removed lots of this->'s to reduce file size and bc it's nicer
* Removed default ESP keybind
* Changed Velocity description
* Added IntSlider to EditionFaker

### CODE REVIEW

**Comment by Intoprelised** [comment]
45 files changed no way any of us forks can redo this lol, i think its good tho

**Comment by Imrglop** [comment]
how does that reduce the filesize, it will compile to the same thing, also its more readable imo

**Comment by Founderroni** [comment]
> 45 files changed no way any of us forks can redo this lol, i think its good tho

takes like 20mins

File: Horion/Module/Modules/EditionFaker.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "EditionFaker.h"
 
-EditionFaker::EditionFaker() : IModule(0, Category::MISC, "Fakes your device to android") {
+EditionFaker::EditionFaker() : IModule(0, Category::MISC, "Fakes your device") {
+	registerIntSetting("OS", &osValue, osValue, 0, 11);
```
Comment (riesters) [inline Horion/Module/Modules/EditionFaker.cpp:4]: Enums exist you know

Comment (Intoprelised) [reply Horion/Module/Modules/EditionFaker.cpp:4]: true!

**Comment by richardletshacks** [comment]
Please open seperate pull requests for refactors and additions. While new features can be approved rather easily, refactors may require more discussion.

**Pull request closed without merging**